app/models/user.rb

```diff
@@ -14,6 +16,8 @@ def set_admin
   end
 
   scope :pending, -> { where.not(invitation_token: true) }
+  # User
+  scope :accepted, -> { where(!User.where.not(invitation_token: true)) }
```

Comment:
!User.where.not(invitation_token: true) - double negation, it is not readable

also how does this scope work? why there is a need for nesting, when you can just directly check the invitation_token property?
  scope :accepted, -> { where.not(invitation_token: true)} - isnt that enough?

Reply:
I wanted users who have accepted the invitation. I changed it now in members overview branch to "scope :accepted, -> { where.not(sign_in_count: 0) }".